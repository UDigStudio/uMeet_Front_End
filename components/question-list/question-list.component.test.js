import React from 'react';
import QuestionList from './question-list.component';
import renderer from 'react-test-renderer';

//TODO: Change to describe/it

it('renders the question list correctly', () => {
  const component = renderer.create(
    <QuestionList questions={[{ id: '1', text: 'First question?', active: true }, { id: '2', text: 'Second question?', active: false }]} />
  );
  let tree = component.toJSON(); 
  expect(tree).toMatchSnapshot();
})
