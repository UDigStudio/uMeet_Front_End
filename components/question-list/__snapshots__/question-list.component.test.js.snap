// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the question list correctly 1`] = `<View />`;
